### PR TITLE
Update quantum control, simulators and experiment control libraries

### PR DESCRIPTION
**Pull request by lazyoracle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,6 +218,10 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [QMASM](https://github.com/lanl/qmasm) - A quantum macro assembler for D-Wave's quantum annealers.
 - [Quil](https://arxiv.org/abs/1608.03355) - An open hybrid quantum/classical instruction set currently used by Rigetti. [Parser](https://github.com/rigetticomputing/pyquil/tree/master/pyquil/_parser)
 
+## Quantum Control
```
Comment (markf94) [inline README.md:221]: Could you please sort this category by programming language such that it follows the same pattern as the other sections?

**Review by markf94 — CHANGES_REQUESTED** [review]
This is amazing! Thanks for all these new software packages @lazyoracle! I only had the one comment below

**Comment by lazyoracle** [comment]
I have updated the sub categories. Let me know if any other changes are required @markf94

**Comment by markf94** [comment]
Great thanks!

**Pull request merged**